lib/kinesis_client.py

```diff
@@ -0,0 +1,66 @@
+import boto3
+import os
+import time
+
+from botocore.exceptions import ClientError
+from helpers.log_helper import create_log
+
+
+class KinesisClient:
+    """Client for managing connections to and operations with AWS Kinesis."""
+
+    def __init__(self):
+        self.logger = create_log('kinesis_client')
+        self.kinesis_client = boto3.client(
+            'kinesis',
+            region_name=os.environ['AWS_REGION'],
+            aws_access_key_id=os.environ['AWS_ACCESS_KEY_ID'],
+            aws_secret_access_key=os.environ['AWS_SECRET_ACCESS_KEY']
+        )
+
+    def send_records(self, encoded_records):
+        """
+        Sends list of Avro encoded patron records (represented as byte strings)
+        to Kinesis stream.
+        """
+        for i in range(0, len(encoded_records), int(
+                os.environ['KINESIS_BATCH_SIZE'])):
+            encoded_batch = encoded_records[i:i + int(
+                os.environ['KINESIS_BATCH_SIZE'])]
+            kinesis_records = [{'Data': record, 'PartitionKey':
```

Comment:
Just a readability nit here, took me a second to parse these lines.

Reply:
Moved int(os.environ['KINESIS_BATCH_SIZE']) to it's own variable to make it a little more readable